domain-services/fiscal-event-service/src/main/java/org/egov/service/FiscalEventEnrichmentService.java

```diff
@@ -26,6 +26,37 @@
 
     public void enrichFiscalEventPushPost(FiscalEventRequest fiscalEventRequest) {
         RequestHeader requestHeader = fiscalEventRequest.getRequestHeader();
+<<<<<<< HEAD
```

Comment:
Why is there <<< HEAD here? Does this code run properly with this?

Reply:
resolved